enumerable_project.rb

```diff
@@ -0,0 +1,139 @@
+ Copyright (c) Muhammasd Arslan
```

Comment:
This line is causing an error in the linter. It's best to remove it. After fixing this the linter will be able to check your project, and there are some Rubocop offences to be fixed.